.github/workflows/php-tests.yml

```diff
@@ -60,5 +60,6 @@ jobs:
       - name: PHPUnit
         run: composer test --no-interaction
 
-      - name: PHPUnit multisite
-        run: composer run-script test-multisite --no-interaction
+      # This has been disabled as it's causing 401 issues for some of the rest api tests.
```

Comment:
Multisite tests were causing failures for the rest api, but regular tests worked fine. I suspect it's a setup problem? But what's weird is that custom status works fine so I've disabled this for now.

Reply:
It's weird that this is causing multisite failures only. I think we should at least investigate the root cause of the issue to ensure it's not an actual multisite-related bug in plugin code.

Reply:
I should have added that, I'll be looking at this today. Just wanted to get the PR up in the meantime

Reply:
Noting that, that has been fixed - thanks @alecgeatches